### ISSUE TITLE
TemporalConvolution doesn't support disabling bias 

### ISSUE THREAD

**Issue by jacklanchantin** [torch/nn#947]
SpatialConvolution and Linear do, but not TemporalConvolution

**Comment by paigautam** [comment]
need this feature,...anything to circumvent this?

**Comment by fmassa** [comment]
@paigautam keeping the bias doesn't affect the results if you are training the model with batch normalization. And if you are loading a model that was saved in a different framework, you can always set the bias to zero.

**Comment by paigautam** [comment]
@fmassa Indeed, but unfortunately, I am constrained to NOT use batch normalization, is it possible any other way?
